.pre-commit-config.yaml

```diff
@@ -35,7 +35,7 @@ repos:
         args: [--all, --expand-stars]
         name: prune imports
   - repo: https://github.com/PyCQA/isort
-    rev: 5.10.1
+    rev: 5.12.0
```

Comment:
Question here (to @pcrespov as well): I see this version is now not in sync with the other isort in the repository. Is that not going to cause problems?

Reply:
If desired I can update all the dependencies. It seems more people have this problem nowadays -> it is connected with the new poetry release.

Reply:
yes, let's update together the isort in the repo. this can be a perfect use-case to show you how the update system works

Reply:
Just make sure in https://github.com/ITISFoundation/osparc-simcore/pull/3889 that you have isort 5.12.0, ok? :-)